### DISCUSSION TITLE
How can I delete account from remotelysave.com?

### DISCUSSION THREAD

**Discussion by monkey21259** [remotely-save/remotely-save#760] [Q&A] [answered]
Hello.
I want to know how can I delete my account from website.
Thank you :)

**Comment by fyears** [comment]
hi may you send an email from your account email to remotelysave@gmail.com ?

and may i ask why would you like to delete the account?

**Reply by monkey21259** [reply]
Ok. I'll send an email soon including reason. :) thank you!

**Comment by monkey21259** [comment]
Hi. I already sent an email but that has not arrived to you. (
***@***.***)
[image: image.png]

2024년 7월 22일 (월) 오후 2:05, fyears ***@***.***>님이 작성:

**Reply by fyears** [reply]
hi i have received the email but please wait for the weekend (when i have free time) for the account operation taken effect

**Reply by fyears** [reply, marked as answer]
Hi your account has been deleted.
